lib/n3h-mod-spec/connection-spec.js

```diff
@@ -12,7 +12,7 @@ const ConnectionEvent = createEventSpec({
    */
   conError: (id, error) => {
     type.assert.string(id)
-    type.assert.string(error)
+    type.assert.object(error)
```

Comment:
this was the wrong type

Reply:
Actually we were emitting the wrong type... the event system is trying to be immutable, error instances get mangled into empty objects... fixed the emitter to output a string.